packages/core/src/RIFWallet.ts

```diff
@@ -96,7 +96,9 @@ export class RIFWallet extends Signer implements TypedDataSigner {
           ...transactionRequest,
           gasPrice: overriddenOptions.gasPrice,
           gasLimit: overriddenOptions.gasLimit,
-        }, overriddenOptions.tokenPayment)
+        }, overriddenOptions.tokenPayment,
+        overriddenOptions.pendingTxsCount
+        )
```

Comment:
Comment here. @TravellerOnTheRun is refactoring this library and we need to remember this parameter in that refactor